assets/mails/notifications_plan_changed.mjml

```diff
@@ -0,0 +1,18 @@
+{{define "content"}}
```

Comment:
Can you also update https://github.com/cozy/cozy-stack/blob/master/docs/assets.md#emails ?

Reply:
Yes I saw that it was missing so I added it.